modules/theme/src/definitions/views/admin-portal.less

```diff
@@ -268,4 +239,134 @@
   height: 1px;
 }
 
+.wrap {
+	cursor: pointer;
+	float: left;
+	margin-left: -3px;
+	margin-right: 5px;
+}
+.arrow {
+	cursor: pointer;
+	display: inline-block;
+	height: 18px;
+	position: relative;
+	top: 4px;
+	width: 18px;
+	span {
+		-webkit-transition: all 0.2s ease;
+		background-color: #bfbfbf;
+		display: inline-block;
+		height: 0.1rem;
+		position: absolute;
+		top: 0.5rem;
+		transition: all 0.2s ease;
+		width: 0.75rem;
+		&:first-of-type {
```

Comment:
Need a new line before the `&:first-of-type`

Reply:
fixed with commit 6cb1c41